qwiic_i2c/circuitpython_i2c.py

```diff
@@ -330,7 +330,14 @@ def isDeviceConnected(self, devAddress):
 			self._i2cbus.writeto(devAddress, bytearray())
 			isConnected = True
 		except:
-			pass
+				try:
+					# ESP32 + CircuitPython doesn't like writing an empty bytearray
+					# So we will try another connection test as well:
```

Comment:
Maybe just say "some platforms (eg. ESP32) don't like writing..." since I'm not sure if this problem is exclusive to ESP32.